### PR TITLE
adding docs for API_GetGameLeaderboards

### PR DESCRIPTION
**Pull request by ioslife**
https://github.com/RetroAchievements/RAWeb/pull/2563

### CODE REVIEW

**Comment by wescopeland** [comment]
Looks like there may be a merge conflict here.

**Comment by ioslife** [comment]
> Looks like there may be a merge conflict here.

Yup, knew that was coming. Should be resolved now.

File: docs/public/game-leaderboards.png
Comment (wescopeland) [inline docs/public/game-leaderboards.png:1]: Can't help but notice this is a nearly 2MB image. Possible we can get this down to just the leaderboards section and a few leaderboards in the list?

Comment (ioslife) [reply docs/public/game-leaderboards.png:1]: Donezo

**Review by wescopeland — APPROVED** [review]

**Pull request merged**